steam_wrapper/steam_wrapper.py

```diff
@@ -255,4 +292,5 @@ def main(steam_binary=STEAM_PATH):
     timezone_workaround()
     configure_shared_library_guard()
     enable_discord_rpc()
-    os.execve(steam_binary, [steam_binary] + sys.argv[1:], os.environ)
+    p = subprocess.Popen([steam_binary] + sys.argv[1:])
```

Comment:
@nanonyme @refi64 If we can avoid IO-intensive copying of the whole extension, maybe we should switch back to using `os.execve`?